rosdep/python.yaml

```diff
@@ -6022,6 +6022,15 @@ python3-anytree-pip:
   ubuntu:
     pip:
       packages: [anytree]
+python3-apa102-pi-pip:
+  debian:
+    pip:
+      depends: [python-rpi.gpio-pip]
```

Comment:
Is this a python2 dependency for this python3 package? I noticed in the https://github.com/tinue/apa102-pi#quick-raspberry-pi-setup section that it suggests `sudo apt install -y python3-pip python3-venv python3-rpi.gpio` - so should we also depend on the native package `python3-rpi.gio`?

Reply:
I think you are right. I left it this way because it worked... Probably python3 was downloading required dependency itself in background.

I could add new dependency `python3-rpi.gpio-pip`, but I'm not sure if it is in the scope of this PR.

Reply:
I think if it's an implicit dependency that `pip` will download automatically, then you don't need to specify anything at all. The `depends:` field is for native packages that this `pip` package depends on, basically for cross-package-manager usage.

Reply:
Thank you for detailed description and help. I removed `depends` filed and left it to download dependency itself.